JDBC/DatabasesPractice/src/main/java/com/laba/solvd/database/Parsing/ParsingMain.java

```diff
@@ -0,0 +1,72 @@
+package com.laba.solvd.database.Parsing;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+
+import javax.xml.XMLConstants;
+import javax.xml.namespace.QName;
+import javax.xml.stream.XMLEventReader;
+import javax.xml.stream.XMLInputFactory;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamReader;
+import javax.xml.stream.events.StartElement;
+import javax.xml.stream.events.XMLEvent;
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.validation.Validator;
+
+import org.xml.sax.SAXException;
+
+public class ParsingMain {
+
+	public static void main(String[] args) throws FileNotFoundException, XMLStreamException {
+		String path2 = "src/main/java/com/laba/solvd/database/localDBS/Objects.xsd";
+		String path1 = "src/main/java/com/laba/solvd/database/localDBS/Objects.xml";
+		boolean validity = validateXmlAgainstXsd(path1, path2);
+		System.out.println(validity);
+		parseAndCreateObjects();
+	}
+	
+	public static boolean validateXmlAgainstXsd(String xmlPath, String xsdPath) {
+		SchemaFactory schemaFactory = SchemaFactory.newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI);
+		Schema schema;
+		try {
+			schema = schemaFactory.newSchema(new StreamSource(xsdPath));
+			Validator validator = schema.newValidator();
+			validator.validate(new StreamSource(new File(xmlPath)));
+			return true;
+		} catch (SAXException e) {
+			e.printStackTrace();
+			return false;
+		} catch (FileNotFoundException e) {
+			e.printStackTrace();
+			return false;
+		} catch (IOException e) {
+			e.printStackTrace();
+			return false;

```

Comment:
The requirement for this homework was to validate XML file using XSD schema and assigned parser. I don't see that you have used one of the parsers for validation.